extensions/emoji/src/emoji.tsx

```diff
@@ -106,32 +113,40 @@ export default function Main(): ReactElement {
                   id={`${category.category}${emoji.description}`}
                   icon={emoji.emoji}
                   title={emoji.description.replace(/\b(\w)/g, (s) => s.toUpperCase())}
-                  accessoryTitle={emoji?.shortCode?.join(" / ")}
                   keywords={emoji.shortCode}
                   actions={
                     <ActionPanel>
                       <ActionPanel.Section>
                         <Action.Paste
-                          title="Paste Emoji in Active App"
                           content={emoji.emoji}
                           onPaste={() => {
-                            closeMainWindow();
-                            popToRoot();
```

Comment:
I noticed this is intentionally added in https://github.com/raycast/extensions/pull/872, here I'm removing it because:

- This follows the built-in Emoji search extension
- It allows you to copy multiple emojis
- If a user prefer popping to root, there is a Raycast settings for this